tests/tsutil.py

```diff
@@ -28,6 +28,19 @@
 import tsinfer
 
 
+def mark_mutation_times_unknown(ts):
+    """
+    Recent msprime versions provide times for mutations, which cannot be estimated by
+    tsinfer
+    """
+    tables = ts.dump_tables()
+    try:
+        tables.mutations.time = np.full_like(tables.mutations.time, tskit.UNKNOWN_TIME)
+    except AttributeError:
```

Comment:
We're not interested in supporting older versions of msprime. We can add msprime>=1.0 to requirements.txt